django/contrib/gis/db/backends/spatialite/operations.py

```diff
@@ -31,6 +31,13 @@ def as_sql(self, connection, lookup, template_params, sql_params):
         return super().as_sql(connection, lookup, template_params, sql_params)
 
 
+class SpatialiteNullCheckOperator(SpatialOperator):
+    def as_sql(self, connection, lookup, template_params, sql_params):
+        sql_template = '%s > 0' % (self.sql_template or lookup.sql_template or self.default_template)
```

Comment:
It might be worth adding some sort of hook to the superclass so that so much copy/paste from the parent `as_sql()` is required.